haskell.nix/application/.github/workflows/check.yml

```diff
@@ -1,5 +1,10 @@
 name: Nix flake check
-on: push
+on:
+  pull_request:
+  push:
+    # If your is default branch is 'main' comment out the next line and uncomment the line following it
```

Comment:
Why "comment out"? I think deleting is better, this `# branches: master` seems useless in the final version.

Reply:
Fair enough, updated